cilium-cli/connectivity/builder/client_egress_l7_tls_headers.go

```diff
@@ -19,7 +19,7 @@ func (t clientEgressL7TlsHeaders) build(ct *check.ConnectivityTest, templates ma
 }
 
 func clientEgressL7TlsHeadersTest(ct *check.ConnectivityTest, templates map[string]string, portRanges bool) {
-	testName := "client-egress-l7-tls-headers"
+	testName := "seq-client-egress-l7-tls-headers"
```

Comment:
@youngnick how about this test?

Reply:
Damn, trying to multitask too hard this week.